proto/frequenz/api/microgrid/common.proto

```diff
@@ -145,91 +148,94 @@ message AC {
   // AC metrics of a single phase.
   message ACPhase {
     // The AC voltage between the line and the neutral wire.
-    // In Volts (V).
+    // In Volt (V).
     Metric voltage = 1;
 
     // AC current.
     // +ve current means consumption, away from the grid.
     // -ve current means supply into the grid.
-    // In Amperes (A).
+    // In Ampere (A).
     Metric current = 2;
 
     // AC active power.
     // +ve power means consumption, away from the grid.
     // -ve power means supply into the grid.
-    // In Watts (W).
+    // In Watt (W).
     Metric power_active = 3;
 
     // AC reactive power.
     // +ve power means inductive (leading).
     // -ve power means capacitive (lagging).
-    // In VArh.
+    // In Volt-Ampere reactive hour (VArh).
     Metric power_reactive = 4;
 
     // The total apparent energy. A Positive value represents the net apparent
     // energy supplied to the grid connection, and vice versa.
-    // In VAh.
+    // In Volt-Ampere-hour (VAh).
     Metric energy_apparent = 5;
 
     // The total active energy counters for the underlying component's
     // consumption and supply.
-    // In Watthours (Wh).
+    // In Watt-hour (Wh).
     ActiveEnergy energy_active = 6;
 
     // The total reactive energy counters for the underlying component's
     // capacitive and inductive energy values.
-    // In VArh.
+    // In Volt-Ampere reactive hour (VArh).
+    // FIXME: ReactiveEnergy says it's in VAh
     ReactiveEnergy energy_reactive = 7;
 
     // Harmonics of the instantaneous active power at the component.
     Harmonics harmonics_power_active = 8;
 
     // Total harmonic distortion
     // of the instantaneous active power at the component.
+    // FIXME: Unit?
```

Comment:
Need feedback here.

Reply:
It's in percent (%).

Harmonics are also in percent (%).